app/Providers/AppServiceProvider.php

```diff
@@ -24,14 +23,6 @@ public function boot()
      */
     public function register()
     {
-        // @TODO: This should be registered in Gateway's service provider!
-        $this->app->singleton(Blink::class, function () {
-            return new Blink(config('services.blink'));
-        });
-
-        // Ensure that debugbar doesn't register on production.
```

Comment:
Will this register on production now?

Reply:
It's registered using Laravel's [package discovery](https://laravel.com/docs/5.5/packages#package-discovery) and marked as a `devDependency`, so it shouldn't. Let's double-check after next deploy just in case!